overlay/master.cpp

```diff
@@ -270,12 +270,14 @@ struct Overlay
       const Option<Network>& _network,
       const Option<Network>& _network6,
       const Option<uint8_t> _prefix,
-      const Option<uint8_t> _prefix6)
+      const Option<uint8_t> _prefix6,
+      const bool& _enabled)
     : name(_name),
     network(_network),
     network6(_network6),
     prefix(_prefix),
-    prefix6(_prefix6)
+    prefix6(_prefix6),
+    enabled(_enabled)
```

Comment:
How about `enabled_ip6` or `enabled6` instead of just `enabled`?

Reply:
I would prefer to make this fix not too specific to the ipv6 issue and be able to disable any overlays in general, at least on new agents.